src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -702,21 +782,8 @@ def _visit_shift(self, node: gtir.FunCall) -> IteratorExpr:
                 it, offset_provider, offset_table_node, offset_expr
             )
 
-    def visit_FunCall(self, node: gtir.FunCall) -> IteratorExpr | ValueExpr:
-        if cpm.is_call_to(node, "deref"):
-            return self._visit_deref(node)
-
-        elif cpm.is_call_to(node, "neighbors"):
-            return self._visit_neighbors(node)
-
-        elif cpm.is_applied_reduce(node):
-            return self._visit_reduce(node)
-
-        elif cpm.is_applied_shift(node):
-            return self._visit_shift(node)
-
-        else:
-            assert isinstance(node.fun, gtir.SymRef)
+    def _visit_builtin(self, node: gtir.FunCall) -> DataExpr:
```

Comment:
A reference to the switch is missing here, the name is to general.
Something along the line "This function is called if `visit_FunCall()` encounters a builtin function".
This is mostly because of its signature that begs the question "why is this function not called `visit_FunCall()`?!?".
And is the name actually correct?
I mean `dref()` is also a build in, but not handled by this function.